docs/useful_links.rst

```diff
@@ -0,0 +1,19 @@
+Useful links
+============
+
+**Related projects, that build abstractions on top of virtualenv**
+
+* `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_ - a useful set of scripts for creating and deleting virtual environments
+* `tox <https://tox.readthedocs.io/en/latest/>`_ - a generic virtualenv management and test automation command line tool, driven by a ``tox.ini``
+  configuration file
+* `nox <https://nox.thea.codes/en/stable/>`_ - a tool that automates testing in multiple Python environments, similar to tox,
+  driven by a ``noxfile.py`` configuration file
+
+**Tutorials**
+
+* `Corey Schafer tutorial <https://www.youtube.com/watch?v=N5vscPTWKOk>`_ on how to use it
+* `Using virtualenv with mod_wsgi <http://code.google.com/p/modwsgi/wiki/VirtualEnvironments>`_
+
+**Presenting how the package works from within**
+
+* `Bernat Gabor: status quo of virtual environments <https://www.youtube.com/watch?v=o1Vue9CWRxU>`_
```

Comment:
Add back Carl Meyers video; it's an interesting case study on how things were in the past.

Reply:
Given the video's quality and age, I am much more inclined to write a document how things were before in the past.

Reply:
Given the quality and the age of the video, I'd be inclined to write a document on how things were in the past.